odl/operator/operator.py

```diff
@@ -668,6 +668,9 @@ def __str__(self):
     # vector properly
     __array_priority__ = 2000000.0
 
+    def __neg__(self):
```

Comment:
It is. I added it along time ago and must have overlooked it once I did a rebase.